world/zonelist.h

```diff
@@ -74,7 +74,10 @@ class ZSList
 	const std::list<std::unique_ptr<ZoneServer>> &getZoneServerList() const;
 	inline uint32_t GetServerListCount() { return zone_server_list.size(); }
 	void SendServerReload(ServerReload::Type type, uchar *packet = nullptr);
+	std::mutex m_queued_reloads_mutex;
```

Comment:
This doesn't play nice with Windows compile, it wants `#include <mutex>`

Reply:
Addressed in #4896